selftests/functional/test_skiptests.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import sys
+import tempfile
+
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
+
+from avocado.core import exit_codes
+from avocado.utils import script
+from avocado.utils import process
+
+basedir = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+basedir = os.path.abspath(basedir)
+
+
+AVOCADO_TEST_SKIP_DECORATORS = """#!/usr/bin/env python
+from avocado import Test
+
+class AvocadoSkipTests(Test):
+
+    @Test.skipIf(True, 'Skipped due to the True condition')
+    def test1(self):
+        pass
+
+    @Test.skipUnless(False, 'Skipped due to the False condition')
+    def test2(self):
+        pass
+"""
```

Comment:
Maybe this test could also include conditions that are evaluated using user written libraries in the same directory as the test. This seems to be always a tricky subject, given the way we handle module imports.